app/views/course/achievements/_form.html.slim

```diff
@@ -5,6 +5,10 @@
 
   = f.input :description
 
+  = f.label t('.badge')
+  div.achievement-form = display_achievement_badge(@achievement)
+  = f.file_field :badge, class: 'upload-btn'
```

Comment:
OK, find where all the upload buttons are, we should style them all the same.

I rather not style components directly because that makes the UI inconsistent.